cmd/k8s-device-plugin/main.go

```diff
@@ -129,14 +130,39 @@ func (p *Plugin) ListAndWatch(e *pluginapi.Empty, s pluginapi.DevicePlugin_ListA
 
 	devs := make([]*pluginapi.Device, len(p.AMDGPUs))
 
+	var hw hwloc.Hwloc
+	hw.Init()
+
```

Comment:
Not needed in this case, but if the code below gets more complicated then perhaps add a "defer hw.Destroy()" here rather than the explicit call at 164.